src/Microsoft.DotNet.Interactive.Documents/InteractiveDocument.cs

```diff
@@ -7,11 +7,11 @@ namespace Microsoft.DotNet.Interactive.Documents
 {
     public partial class InteractiveDocument
     {
-        public InteractiveDocumentElement[] Elements { get; }
+        public InteractiveDocumentElement[] Elements { get; set; }
```

Comment:
Let's make this a `List<InteractiveDocumentElement>` and remove the setter. The property shouldn't be allowed to be null.